Vostok.ClusterConfig.Core/Parsers/FileParserSettings.cs

```diff
@@ -24,5 +24,17 @@ internal class FileParserSettings
         };
 
         public int MaximumFileSize { get; set; } = 1024 * 1024;
+
+        public Dictionary<string, long> MaximumFileSizeZoneOverloads { get; set; } = new Dictionary<string, long>
+        {
+            ["default"] = 200 * 1024
+        };
+
+        public Dictionary<string, long> MaximumFileSizeFileNameOverloads { get; set; } = new Dictionary<string, long>
+        {
+            ["innBlackList"] = 500 * 1024,
+            ["__file_size_test_1K__"] = 1 * 1024,
```

Comment:
I suggest moving these test entries to the respective tests instead of widely used defailts.